views/backend/dashboard.php

```diff
@@ -47,55 +43,48 @@
     </thead>
     <tbody>
 
-        <?php   
+        <?php  
+        $count = 0;
         if(!empty($comments)) {
-            foreach ($comments as $comment) {
-
-                if(filter_has_var(INPUT_GET, 'delete')){
-                    $comment = $model_comment->delete_comment();
-                }
-                
-                if(filter_has_var(INPUT_GET, 'approve')){
-                    $comment = $model_comment->approve_comment();
-                }
+            foreach ($comments as $comment) {  
         ?>
         
         <div>
-                <tr id="commentaire_<?=$comment->id?>">
-                    <td><?= $comment->title ?></td>
-                    <td><?= substr($comment->comment, 0, 100); ?>...</td>
+                <tr id="commentaire_<?= $comment->getId() ?>">
+                    <td><?= $posts[$count]->getTitle() ?></td>
+                    <td><?= substr($comment->getComment(), 0, 100); ?></td>
                     <td>
 
-                        <a href="index.php?page=dashboard&approve=<?= $comment->id ?>" type="submit" 
+                        <a href="index.php?page=dashboard&approve=<?= $comment->getId() ?>" type="submit" 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$comment'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)